app/ui/legacy/src/main/res/layout/recipient_alternate_item.xml

```diff
@@ -143,6 +143,15 @@
             tools:visibility="visible"
             />
 
+        <ImageView
+            android:id="@+id/button_copy_email_address"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="13dp"
+            android:background="?attr/selectableItemBackground"
```

Comment:
Using `?attr/controlBackground` will use a circular background that looks better with action icons.